orm/src/test/java/org/hibernate/search/test/analyzer/inheritance/AnalyzerInheritanceTest.java

```diff
@@ -53,12 +56,12 @@ public void testBySearch() throws Exception {
 		tx = s.beginTransaction();
 
 
-		QueryParser parser = new QueryParser( "name", s.getSearchFactory().getAnalyzer( SubClass.class ) );
-		org.apache.lucene.search.Query luceneQuery = parser.parse( "name:Proca\u00EFne" );
+		QueryParser parser = new QueryParser( "name", TestConstants.keywordAnalyzer );
+		org.apache.lucene.search.Query luceneQuery = parser.parse( "name:John" );
 		FullTextQuery query = s.createFullTextQuery( luceneQuery, SubClass.class );
 		assertEquals( 1, query.getResultSize() );
 
-		luceneQuery = parser.parse( "name:Procaine" );
+		luceneQuery = parser.parse( "name:Doe" );
```

Comment:
Thing is, we can only use a built-in Lucene analyzer, unless we want to alter the test another way, by using analyzer definitions instead of an analyzer implementation.
I changed this to use Turkish diacritics. `StandardAnalyzer` seems to fail at removing those and the turkish analyzer is built-in.